app/controllers/staff/activities_controller.rb

```diff
@@ -3,11 +3,15 @@
 class Staff::ActivitiesController < Staff::BaseController
   include Secured
   after_action :verify_authorized, except: [:index, :historic]
-  after_action :verify_policy_scoped, only: [:index, :historic]
+  after_action :skip_policy_scope, only: [:index, :historic]
```

Comment:
What do you think about adding one sentance to the commit message about the fact you have to skip the policy scope because we have `verify_policy_scoped` in the base controller so all our controllers have it switched on, I think we forget about this kind of inheritance on controllers (e.g. `include Secured`) sometimes?

Reply:
Cool, done! 👍